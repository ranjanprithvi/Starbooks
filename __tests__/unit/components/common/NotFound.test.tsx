import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import NotFound from "../../../../src/components/common/NotFound";
import React from "react";

test("Renders the NotFound page", () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
});
