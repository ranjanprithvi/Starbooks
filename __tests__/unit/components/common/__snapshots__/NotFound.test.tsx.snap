// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the NotFound page 1`] = `
<div>
  NotFound
</div>
`;
